DuckDuckGo/AppDelegate.swift

```diff
@@ -82,13 +82,22 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     }
     
     private func startOnboardingFlowIfNotSeenBefore() {
+        
         var settings = TutorialSettings()
         if !settings.hasSeenOnboarding {
+            startMigration()
```

Comment:
Suggestions on where to put it?  I figured on-boarding was good because it would be complete by the time the user had got through it and already has logic for only doing this once.